### DISCUSSION TITLE
Not able to upload my proof

### DISCUSSION THREAD

**Discussion by anishaman** [community/community#36620] [GitHub Education]
Dear Team,
My ID card of the university contains name on one side and date on other side, so i need to make a combined pdf and upload it. but while filling up my application, there is only option of taking a photo via the laptop camera and there is no option for uploading any proof.

Kindly look into this and resolve this.

**Comment by Neodevils** [comment]
You can upload it. You don't really have to take a picture and upload it.

I did it with taking a screenshot, then uploaded it.

**Reply by anishaman** [reply]
But there is no upload button to do so.  Check the screenshot of the page i am getting.(attached)
![Uploading Screenshot 2022-07-30 154643.png…]()

**Comment by anishaman** [comment]
But there is no upload button to do so.  Check the screenshot of the page i
am getting.(attached)

**Comment by vykouzlith2fb3b** [comment]
Why is my document not uploading?
Your file may be corrupted,[marine equipment dubai](https://www.mazuzee.com/), which means there are metadata issues in the file that occurred during original document generation, scanning, or some other conversion process, has produced an invalid PDF (i.e., one that doesn't conform to PDF structure standards).

**Comment by ossanna16** [comment]
Hey there @anishaman and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by AndreiZoltan** [comment]
I have the same problem. 
No matter how I resize the picture, it still throws the error: "File is too small".

**Reply by carloscapella** [reply]
Same error, can not upload proof , is there another way to upload it ?
